InnerEye/Azure/azure_runner.py

```diff
@@ -452,3 +486,33 @@ def run_duration_string_to_seconds(s: str) -> Optional[int]:
     else:
         raise ArgumentError("s", f"Invalid suffix: Must be one of 's', 'm', 'h', 'd', but got: {s}")
     return int(float(s[:-1]) * multiplier)
+
+
+def set_environment_variables_for_multi_node() -> None:
+    """
+    Sets the environment variables that PyTorch Lightning needs for multi-node training.
+    """
+    az_master_node = "AZ_BATCHAI_MPI_MASTER_NODE"
+    master_addr = "MASTER_ADDR"
+    master_ip = "MASTER_IP"
+    master_port = "MASTER_PORT"
+    world_rank = "OMPI_COMM_WORLD_RANK"
+    node_rank = "NODE_RANK"
+
+    if az_master_node in os.environ:
+        # For AML BATCHAI
+        os.environ[master_addr] = os.environ[az_master_node]
+    elif master_ip in os.environ:
+        # AKS
+        os.environ[master_addr] = os.environ[master_ip]
+    else:
+        logging.info("No settings for the MPI central node found. Assuming that this is a single node training job.")
+        return
+
+    if master_port not in os.environ:
+        os.environ[master_port] = "6105"
```

Comment:
Is this something AzureML? Do they have a constant in their SDK?

Reply:
I did not find any such constant in azureML, nor in Lightning.